pkg/minikube/out/style.go

```diff
@@ -70,6 +70,7 @@ var styles = map[StyleEnum]style{
 	ThumbsUp:      {Prefix: "👍  "},
 	ThumbsDown:    {Prefix: "👎  "},
 	Option:        {Prefix: "    ▪ ", LowPrefix: lowIndent}, // Indented bullet
+	OptionVerify:  {Prefix: "    🔎 ", LowPrefix: lowIndent}, //  Indented verifying  icon
```

Comment:
This doesn't seem like an option - maybe IndentVerify?

Also, remove extra space before icon

Reply:
the extra space seems to be making it work on mac. this icon is weird.